core/datasource/src/main/java/com/tangem/datasource/local/onramp/countries/DefaultOnrampCountriesStore.kt

```diff
@@ -0,0 +1,11 @@
+package com.tangem.datasource.local.onramp.countries
+
+import com.tangem.datasource.local.datastore.core.StringKeyDataStore
+import com.tangem.datasource.local.datastore.core.StringKeyDataStoreDecorator
+import com.tangem.domain.onramp.model.OnrampCountry
+
+internal class DefaultOnrampCountriesStore(
+    dataStore: StringKeyDataStore<List<OnrampCountry>>,
+) : OnrampCountriesStore, StringKeyDataStoreDecorator<String, List<OnrampCountry>>(dataStore) {
```

Comment:
Тут можно без декоратора, так как ключ и так строка:
```suggestion
) : OnrampCountriesStore, StringKeyDataStore<List<OnrampCountry>> by dataStore {
```

Reply:
Где-то вообще был стор, который может без ключа работать, но его выпилили похоже

Reply:
Давай в следующем ПРе, этот вольем пока